### PR TITLE
[K32W] Improve logging

### PR DESCRIPTION
**Pull request by doru91**
#### Problem
* k32w was lacking timestamp logging;
* event type was not printed during a long dispatch time.

#### Change overview
* add a timestamp for each logged line;
* print the event type during a long dispatch time.

#### Testing
* manual testing

### CODE REVIEW

**Review by Damian-Nordic — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Review by bzbarsky-apple — APPROVED** [review]

File: src/platform/K32W/Logging.cpp [hunk 1]
```diff
@@ -21,60 +27,59 @@
...
-            break;
+            categoryString = "U";
         }
+
+        writtenLen = snprintf(buf, bufLen, "[%s]", categoryString);
+        bufLen -= writtenLen;
+        buf += writtenLen;
     }
+
+    writtenLen = snprintf(buf + writtenLen, bufLen, "[%s]", module);
```
Comment (mspang) [inline src/platform/K32W/Logging.cpp:73]: Didn't we already add writtenLen ?

Comment (mspang) [reply src/platform/K32W/Logging.cpp:73]: @bzbarsky-apple @doru91

Comment (bzbarsky-apple) [reply src/platform/K32W/Logging.cpp:73]: Er, yes, so we did.  Good catch!  @doru91 please fix?

Comment (doru91) [reply src/platform/K32W/Logging.cpp:73]: Thanks @mspang @bzbarsky-apple . I opened a new PR: https://github.com/project-chip/connectedhomeip/pull/8471 for fixing this issue as this PR have been merged.

Comment (bzbarsky-apple) [reply src/platform/K32W/Logging.cpp:73]: Fixed in https://github.com/project-chip/connectedhomeip/pull/8471

**Pull request merged**